test/source-updater.test.js

```diff
@@ -1313,3 +1314,38 @@ QUnit[testOrSkip]('audio appends are delayed until video append for the first ap
     assert.ok(!audioAppend, 'audio has not appended yet');
   });
 });
+
+QUnit.test('appendBuffer calls back with QUOTA_EXCEEDED_ERR', function(assert) {
+  assert.expect(2);
+
+  this.sourceUpdater.createSourceBuffers({
+    audio: 'mp4a.40.2',
+    video: 'avc1.4D001E'
+  });
+
+  const videoBuffer = {
+    appendBuffer() {
+      const quotaExceededError = new Error();
+
+      quotaExceededError.code = QUOTA_EXCEEDED_ERR;
+
+      throw quotaExceededError;
```

Comment:
not that we should change this, but the alternative would've been `throw {code: QUOTA_EXCEEDED_ERR};`

Reply:
The `vjsstandard` linter yelled at me for doing that at first (which I was surprised by, but changing to an actual error helped).